package.json

```diff
@@ -79,35 +87,31 @@
     ],
     "devDependencies": {
         "@babel/core": "^7.6.4",
-        "@babel/polyfill": "^7.6.0",
+        "@babel/plugin-proposal-export-default-from": "^7.5.2",
+        "@babel/plugin-proposal-export-namespace-from": "^7.5.2",
```

Comment:
I had to add it which means ``preset-env`` doesn't include it.

Reply:
This proposal was merged into the ES standard 40 days ago (!) That's why it isn't supported by most browsers yet, and `preset-env` with the current target doesn't include it.

I truly think that the effort needed to make proper decisions about this kind of things isn't worth it. You only get a small syntactic advantage, but you are now in charge of knowing why each plugin is needed, what is its impact in things like bundle sizes, and when can it be removed. 

Let me use the TS example once again. If you are developing using TS, you can delegate most of that work to the TS team. You only have to choose your target version, and they are responsible for the due diligence around adopting proposals. This is good because (1) they are a team of language designers and compiler engineers, that can understand the implications of each decision (2) that's their core value offer so they can afford spending time in this things. 

Don't get me wrong, I **love** being up to date with the TC39's work and talking about it. But this is a high stake project which [is in the process of getting more stable thanks to you and Chris](#3070), so I believe we should be more conservative here.

Reply:
> I truly think that the effort needed to make proper decisions about this kind of things isn't worth it. You only get a small syntactic advantage, but you are now in charge of knowing why each plugin is needed, what is its impact in things like bundle sizes, and when can it be removed.

I think you overreacting. Both of those Babel plugins are small, do work and are maintained from the Babel maintainers. It's also not that hard to subscribe to two tc39 proposal repositories to be up to date and to investigate if required.

> Don't get me wrong, I love being up to date with the TC39's work and talking about it. But this is a high stake project which is in the process of getting more stable thanks to you and Chris, so I believe we should be more conservative here.

I had this setup running for a half a year with the 2.x codebase and didn't have any issue with it that’s why I can't understand why such Babel plugins shouldn't be used. I could bet TS is using this proposal as well in the background. :)